detection_rules/ecs.py

```diff
@@ -148,6 +150,28 @@ def get_non_ecs_schema():
     return load_etc_dump('non-ecs-schema.json')
 
 
+@cached
+def get_custom_schemas(stack_version: str):
+    """Load custom schemas if present."""
+    custom_schema_dump = {}
+    stack_schema_map = RULES_CONFIG.stack_schema_map[stack_version]
+
+    for schema, value in stack_schema_map.items():
+        if schema not in ["beats", "ecs", "endgame"]:
+            schema_path = RULES_CONFIG.stack_schema_map_file.parent / value
```

Comment:
I keep going back and forth of wether it should be relative to the custom_dir path (like this) or require it to be an absolute path. Primary reason for absolute over relative - if they want to use our built-in schemas in a customized way. Perhaps we could even do both

* if it is absolute, use it
* if it is not, append to relative path as is

Reply:
Decided to update to support this, I like it :+1: